### ISSUE TITLE
Investigate Nan issue in training

### ISSUE THREAD

**Issue by LiyouZhou** [LiyouZhou/lerobot#3]
In direct action prediction, with memory integrated, the training seem to be unstable:

https://wandb.ai/leothemagnificent-university-of-cambridge/smolvla/runs/mikasa_all_lerobot-smolvla_base_B64_Etrue_Mtrue_S1000_251010_020620?nw=nwuserleothemagnificent

Need to find out,

What is the gradient norm in the inner loop, make sure the learning rate is set appropriately
Print model gradient using wandb.watch(my_model, log='all')

Fine tune gradient clipping and learning rate to make training stable.

The nan seem to come from begining of episodes where the gradient is large, also maybe the inner gradient is too large in this case.

We can make the memory init trainable, also improvement is batch diversity would also help.
